xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h

```diff
@@ -47,7 +47,8 @@ enum StreamType
   STREAM_DATA, // data stream
   STREAM_SUBTITLE, // subtitle stream
   STREAM_TELETEXT, // Teletext data stream
-  STREAM_RADIO_RDS // Radio RDS data stream
+  STREAM_RADIO_RDS, // Radio RDS data stream
+  STREAM_ENUM_COUNT
```

Comment:
Is this right here? All the other enum entries refer to a concrete type, what is `STREAM_ENUM_COUNT`?